specification/workloads/resource-manager/Microsoft.Workloads/preview/2021-12-01-preview/monitors.json

```diff
@@ -0,0 +1,1190 @@
+        "x-ms-examples": {
+          "Update Tags field of a SAP monitor": {
+            "$ref": "./examples/workloadmonitor/monitors_PatchTags.json"
+          },
+          "Delete Tags field of a SAP monitor": {
+            "$ref": "./examples/workloadmonitor/monitors_PatchTags_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/monitorNameParameter"
+          },
+          {
+            "$ref": "#/parameters/TagsParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Monitor"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Workloads/monitors/{monitorName}/providerInstances": {
+      "get": {
+        "tags": [
+          "ProviderInstances"
+        ],
+        "operationId": "ProviderInstances_List",
+        "summary": "Gets a list of provider instances in the specified SAP monitor.",
+        "description": "Gets a list of provider instances in the specified SAP monitor. The operations returns various properties of each provider instances.",
+        "x-ms-examples": {
+          "List all SAP Monitors providers in a subscription": {
+            "$ref": "./examples/workloadmonitor/ProviderInstances_List.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/monitorNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ProviderInstanceListResult"
```

Comment:
>ProviderInstanceListResult [](http://example.com/codeflow?start=37&length=26)

This will contain secret dbPassword?  Get should not return secret, consider use POST to retrieve secret, see Secrets in [https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html)

Reply:
No, password is a secret property hence it can't be read. it is marked as x-ms-secret in monitors.json